docs/source/posts/2021/2021-06-08-week-1-sajag.rst

```diff
@@ -0,0 +1,41 @@
+Welcome to my GSoC Blog!
+========================
+
+.. post:: June 8 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Hi all! 
+I'm Sajag Swami, a sophomore at Indian Institute of Technology, Roorkee. This summer, I will be working on adding a new functionality to **FURY** which shall enable users to 
+visualise various types of proteins via different representations like Richardson aka Ribbon diagrams and molecular surface diagrams. 
+As a part of my stretch goals, I’ll try to expand protein diagrams via other representations including -
+
+1. Stick
+2. Ball and stick
+3. Wire
+4. Pipes and Planks
+5. Sphere
+
+What did you do during the Community Bonding Period?
+----------------------------------------------------
+I had weekly meetings with my mentors and other core team members. In the first meeting I got acquainted with the team members and learnt about the organisation and its goal/vision.
+In the later meetings we discussed about various representations of proteins and how to go about implementing them in FURY.
+We discussed about various libraries which can be used to parse PDB and PDBx files.
+I made a `document <https://docs.google.com/document/d/1mSoAWyXlLNrCa3hN-hiP35Lj7rURYMk5jFnWZbZp70s>`_ for the same to list pros and cons of using each library. 
```

Comment:
We might need to put this google document in rst and somewhere else.

Reply:
Any suggestions as to where should I put it?